README.md

```diff
@@ -108,8 +108,10 @@ transformer.generate("wrappers").catch(err => {
 
 The main module's default export is a class which you can construct with a few options:
 
-- `implSuffix`: a suffix used, if any, to find files within the source directory based on the IDL file name.
-- `suppressErrors`: set to true to suppress errors during generation.
+- `implSuffix`: a suffix used, if any, to find files within the source directory based on the IDL file name. (default: `""`)
+- `suppressErrors`: set to true to suppress errors during generation. (default: `false`)
+- `processCEReactions`: an optional hook to post-process the generated code for IDL operation and attributes annotated with the [`[CEReaction]`](https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions) extended attributes. The processor expects the following signature `(idl: IdlOperation | IdlAttribute, code: string): string` where `idl` is the parsed [operation member](https://github.com/w3c/webidl2.js#operation-member) and [attribute member](https://github.com/w3c/webidl2.js#attribute-member) and `code` the generated accessor code.
```

Comment:
```suggestion
- `processCEReactions`: an optional hook to post-process the generated code for IDL operation and attributes annotated with the [`[CEReaction]`](https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions) extended attribute. The processor expects the following signature `(idl: IdlOperation | IdlAttribute, code: string): string` where `idl` is the parsed [operation member](https://github.com/w3c/webidl2.js#operation-member) and [attribute member](https://github.com/w3c/webidl2.js#attribute-member) and `code` the generated accessor code.
```

I'd also maybe say a few words on a) the default behavior (pass-through) as well as b) what the `code` parameter actually contains. An example here would be more than sufficient.

Same for `[HTMLConstructor]`.